plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimatorsFactory.kt

```diff
@@ -318,9 +318,11 @@ class CameraAnimatorsFactory internal constructor(mapDelegateProvider: MapDelega
     // w₀: Initial visible span, measured in pixels at the initial scale.
     // Known henceforth as a <i>screenful</i>.
 
+    val size = mapTransformDelegate.getMapOptions().size!!
```

Comment:
we were using `mapTransformDelegate.getSize()` and it seems we don't have to use `!!`, is there any difference with these two methods?

Reply:
We have because size is for some reason marked `Nullable`.
There is no difference I assume, the one we used before was the one I exposed myself (most likely I just did not notice that we already have size as part of `MapOptions` or it was not exposed at that point there).
But now it's more logical to use the one from `MapOptions` because I need pixel ratio as well.